qiskit/providers/ibmq/exceptions.py

```diff
@@ -27,19 +27,51 @@ class IBMQAccountError(IBMQError):
     pass
 
 
-class IBMQApiUrlError(IBMQAccountError):
```

Comment:
Removing this makes it an incompatible change. It's probably fine since it's unlikely anyone is trapping `IBMQApiUrlError`, but it should be mentioned in the changelog.